README.md

```diff
@@ -21,6 +21,25 @@ The `push-image` action builds a Docker image and pushes it to an appropriate Go
     labels: project=myproject
 ```
 
+#### How to push images to ECR
+
+to push images to ECR you need to provide an ECR `dockerRegistry` and setup few environment variable. For your image to be deployable via slipstream it is mandatory to set the `release` field to `true` otherwise your image will be considered as a development image.
```

Comment:
```suggestion
To push images to ECR you need to provide an ECR `dockerRegistry` and setup a few environment variables. For your image to be deployable via slipstream it is mandatory to set the `release` field to `true` otherwise your image will be considered as a development image.
```